gateleen-routing/src/main/java/org/swisspush/gateleen/routing/Forwarder.java

```diff
@@ -315,6 +315,7 @@ private void installExceptionHandler(final HttpServerRequest req, final String t
                 error("Timeout", req, targetUri);
                 respondError(req, StatusCode.TIMEOUT);
             } else {
+                LOG.warn("Failed to '{} {}'", req.method(), targetUri, exception);
```

Comment:
`exception` parameter is newer used when only having two `{}` placeholders. Or do I miss something?

Reply:
[how-to-log-exception-and-message-with-placeholders-with-slf4j](https://stackoverflow.com/q/5951209/4415884)

Reply:
Nice. They did a good job hiding this in the documentation ;-)